src/toxic_constants.h

```diff
@@ -89,4 +91,30 @@ typedef enum _FATAL_ERRS {
     FATALERR_CURSES = -14,          /* Unrecoverable Ncurses error */
 } FATAL_ERRS;
 
+typedef struct Run_Options {
```

Comment:
This feels pretty out of place in constants.h, but I also don't have any real suggestions for a better home other than maybe a new run_options.h.

Reply:
Yeah wasn't sure about that one. Done.